src/ServiceControl.Monitoring.AcceptanceTests/Tests/When_having_raw_data_provided.cs

```diff
@@ -0,0 +1,137 @@
+﻿namespace NServiceBus.Metrics.AcceptanceTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Net.Http;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using AcceptanceTesting.Customization;
+    using Features;
+    using global::Newtonsoft.Json;
+    using global::Newtonsoft.Json.Linq;
+    using NServiceBus.AcceptanceTests;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NUnit.Framework;
+    using ObjectBuilder;
+    using ServiceControl.Monitoring;
+    using ServiceControl.Monitoring.Http;
+
+    public class When_having_raw_data_provided : NServiceBusAcceptanceTest
```

Comment:
I find it hard to understand this test, but I think modifying the name slightly should solve the issue for me. e.g.

When_having_raw_data_provided  => Endpoint_using_metrics_provider
Should_report_via_http => Should_report_metrics_via_http